### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by Mark7even** [Botcoins/issue-tracker#40]
Bot went offline and never came back.

**Comment by Thunder33345** [comment]
@Mark7even Yes this project is dead, the bot wont be coming on due to the api we used changed
kinda sucks but oh well
